federation/pkg/federation-controller/sync/controller.go

```diff
@@ -273,10 +273,12 @@ func (s *FederationSyncController) reconcileOnClusterChange() {
 	}
 }
 
-func (s *FederationSyncController) reconcile(namespacedName types.NamespacedName) {
+func (s *FederationSyncController) reconcile(namespacedName types.NamespacedName) (error, map[string]bool) {
+	reconcileResult := make(map[string]bool, 0)
```

Comment:
Based on the name, I was expecting reconcileResult to store the result of whether the reconcile was successful in each cluster or not. But looks like it is storing if the object needs to be reconciled in a cluster? Is that right?

If yes, then is `shouldReconcile` a better name?